utopia-remix/app/routes/projects.tsx

```diff
@@ -852,11 +837,16 @@ const ProjectRow = React.memo(
 ProjectRow.displayName = 'ProjectRow'
 
 const ProjectCardActions = React.memo(({ project }: { project: ProjectWithoutContent }) => {
+  const [sortMenuOpen, setSortMenuOpen] = React.useState(false)
+
   return (
     <div style={{ display: 'flex', alignItems: 'center', gap: 5 }}>
-      <DropdownMenuRoot>
+      <DropdownMenuRoot onOpenChange={() => setSortMenuOpen(!sortMenuOpen)}>
```

Comment:
This should probably be wrapped in a `React.useCallback` function to prevent unnecessary re-renders.